addons/core/translations/form/en-us.yaml

```diff
@@ -149,3 +149,9 @@ ip_addresses:
 sync-interval:
   label: Sync Interval
   help: Number of seconds between the time Boundary syncs hosts in this set.
+guided:
```

Comment:
While the path select is designed as a form, I'm not sure it makes sense to think of these as form fields.  Let's group these under the quick start onboarding translations.

Reply:
Moved.